sklearn/utils/__init__.py

```diff
@@ -589,3 +589,34 @@ def is_scalar_nan(x):
     # Redondant np.floating is needed because numbers can't match np.float32
     # in python 2.
     return bool(isinstance(x, (numbers.Real, np.floating)) and np.isnan(x))
+
+
+def _init_arpack_v0(size, random_state):
```

Comment:
Not sure there is the best place for it, but I couldn't find a better one in `sklearn.utils`